src/components/Page3.jsx

```
@@ -1,14 +1,78 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { TextField, Button } from "@mui/material";
 import CheckCircleOutlineIcon from "@mui/icons-material/CheckCircleOutline";
```

ESLint

[error] 'CheckCircleOutlineIcon' is defined but never used.

(no-unused-vars)